Makefile

```diff
@@ -72,6 +72,7 @@ IPV6_SUPPORT := $(or ${IPV6_SUPPORT}, True)
 MIRROR_REG_CA_FILE = mirror_ca.crt
 REGISTRIES_FILE_PATH = registries.conf
 MIRROR_REGISTRY_SUPPORT := $(or ${MIRROR_REGISTRY_SUPPORT},False)
+OCP_REQUIREMENTS := $(or ${OCP_REQUIREMENTS}, $(shell cat data/default_ocp_requirements.json | tr -d "\n\t "))
```

Comment:
Could we change all namings to `HW_REQUIREMENTS` so it would be more flexible? Is that possible to hold the OLM data as well over there?

Reply:
I will change it to `HW_REQUIREMENTS`.